src/Subscriptions/StorageManager.php

```diff
@@ -110,8 +111,45 @@ public function storeSubscriber(Subscriber $subscriber, string $topic): void
         }
     }
 
+    /**
+     * Delete subscriber.
+     *
+     * @param  string  $channel
+     * @return \Nuwave\Lighthouse\Subscriptions\Subscriber|null
+     */
     public function deleteSubscriber(string $channel): ?Subscriber
     {
-        return $this->cache->pull(self::SUBSCRIBER_KEY.".{$channel}");
+        $key = self::SUBSCRIBER_KEY.".{$channel}";
+
+        if ($this->cache->has($key)) {
```

Comment:
Do not use `has()`, see https://github.com/nuwave/lighthouse/pull/1290

Reply:
Got it.